app/services/irs_attempts_api/tracker.rb

```diff
@@ -51,12 +51,6 @@ def track_event(event_type, metadata = {})
         )
       end
```

Comment:
I think we could also remove the `analytics.irs_attempts_api_event_metadata` and not even build the event here, just let it all be nil